addons/mail/static/src/core/public_web/discuss_sidebar.js

```diff
@@ -6,6 +6,7 @@ import { _t } from "@web/core/l10n/translation";
 
 import { registry } from "@web/core/registry";
 import { useService } from "@web/core/utils/hooks";
+import { UserStatusPanel } from "../web/user_status_panel";
```

Comment:
You are importing from a file that is only loaded on the web bundle in a public_web file. That’s why the public page tests are failing. If you move the panel to the public page, consider the guests. I'm not sure if we plan to enable this feature for guests as well (I guess we do since you used controllers), but it would make the diff bigger, so we might want to merge the first part first and hide the panel for guests in the meantime.

Reply:
Will enable it for guests in a later PR.

It could be the place for the guest renaming as well.